library/Haste/Form/Form.php

```diff
@@ -478,7 +478,17 @@ function($varValue, \Widget $objWidget, Form $objForm) use ($arrDca, $strName) {
             );
         }
 
-        $arrDca = $strClass::getAttributesFromDca($arrDca, $arrDca['name'], $arrDca['value']);
+        $objDca = null;
+        $strTable = '';
+
+        if (null !== ($objModel = $this->getBoundModel())) {
+            $objDca = new \Contao\DC_Table($objModel->getTable());
```

Comment:
I don't think this is correct. What if I replaced the table with a different DC? I guess this would need to be loaded from the `table` setting in the DCA, no?

Reply:
Fair point, changed now.